include/miral/miral/window_manager_tools.h

```diff
@@ -156,6 +156,10 @@ class WindowManagerTools
     /// make the prev surface active within the active application
     void focus_prev_within_application();
 
+    /// Returns true if the application is focusable, otherwise false.
+    /// \remark Since MirAL 3.10
+    auto can_focus_application(Application application) const -> bool;
```

Comment:
I'm not sure we need this function: it is only used in one place, and that place probably belongs in `WindowManagerTools`